cmd-record.c

```diff
@@ -352,9 +354,46 @@ static int fill_file_header(struct opts *opts, int status, struct rusage *rusage
 	if (write(fd, &hdr, sizeof(hdr)) != (int)sizeof(hdr))
 		pr_err("writing header info failed");
 
+	char *orig_args = opts->args;
+	char *orig_retval = opts->retval;
```

Comment:
I prefer declaring variables before any statement like in (old) C syntax.

Reply:
done!